core/services/pipeline/mocks/orm.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery 2.7.4. DO NOT EDIT.
+// Code generated by mockery v2.7.5. DO NOT EDIT.
```

Comment:
Your mockery has version numbers in a different format - `vX.Y.Z` instead of `X.Y.Z`

Reply:
Mockery has really bad versioning, it expects a generator to be ran before building. As a result, installing from homebrew vs `go install` vs various other packages return different versions: https://github.com/vektra/mockery/issues/381

Mine for example always returns `v0.0.0-dev`

Reply:
:sob: